templates/nodejs/install.erb

```diff
@@ -8,7 +8,7 @@ else
     FILENAME='linux-x64.tar.gz'
 fi
 
-LATEST_NODE=$(curl 'http://nodejs.org/dist/latest/SHASUMS.txt' | grep "${FILENAME}" | awk '{ print $2 }')
+LATEST_NODE=$(curl 'http://nodejs.org/dist/latest/SHASUMS256.txt' | grep "${FILENAME}" | awk '{ print $2 }')
```

Comment:
Changing the file does not solve anything. You get a list of names and the code does nothing.

```
[10:35 AM]-[root@packer-virtualbox-iso-1422601639]-[/.puphpet-stuff]
# curl 'http://nodejs.org/dist/latest/SHASUMS256.txt' | grep "${FILENAME}" | awk '{ print $2 }'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2674    0  2674    0     0  19816      0 --:--:-- --:--:-- --:--:-- 19955
node-v4.1.0-darwin-x64.tar.gz
node-v4.1.0-darwin-x64.tar.xz
node-v4.1.0-headers.tar.gz
node-v4.1.0-headers.tar.xz
node-v4.1.0-linux-arm64.tar.gz
node-v4.1.0-linux-arm64.tar.xz
node-v4.1.0-linux-armv6l.tar.gz
node-v4.1.0-linux-armv6l.tar.xz
node-v4.1.0-linux-armv7l.tar.gz
node-v4.1.0-linux-armv7l.tar.xz
node-v4.1.0-linux-armv7.tar.gz
node-v4.1.0-linux-armv7.tar.xz
node-v4.1.0-linux-x64.tar.gz
node-v4.1.0-linux-x64.tar.xz
node-v4.1.0-linux-x86.tar.gz
node-v4.1.0-linux-x86.tar.xz
node-v4.1.0.pkg
node-v4.1.0-sunos-x64.tar.gz
node-v4.1.0-sunos-x64.tar.xz
node-v4.1.0-sunos-x86.tar.gz
node-v4.1.0-sunos-x86.tar.xz
node-v4.1.0.tar.gz
node-v4.1.0.tar.xz
node-v4.1.0-x64.msi
node-v4.1.0-x86.msi
win-x64/node.exe
win-x64/node.lib
win-x86/node.exe
win-x86/node.lib
```